.analysis_options

```diff
@@ -1,4 +1,6 @@
 analyzer:
+  language:
+    enableAssertInitializer: true
```

Comment:
Is this necessary? I don't think we want to enable experimental language features that we might later have to remove. Can we enable the feature just for the test? (I'm assuming that's why this is here.)